src/translations.js

```diff
@@ -158,6 +163,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       sk: "Účasti",
       tr: "Katkı Verildi",
       pl: "Udziały",
+      uz: "'Contribute'ga qatnashdi",
```

Comment:
Here Google translate gives me `hissa qo'shgan`.

Reply:
Depending on where the word is used in the program, it may be possible to use the word you suggested.

Where is this word used?

Reply:
It is used on the stats card:

![image](https://user-images.githubusercontent.com/17570430/154437749-86e234a9-debe-496a-a6f9-8074b7d73fa7.png)

Reply:
@MrMuminov Great! Can you apply these two changes `Wakatime statistikasi` and `Hissa qoʻshgan`. After that I can merge the pr.